Source/IntegrationTests/TestFiles/LitTests/LitTest/verification/progress.dfy.expect

```diff
@@ -0,0 +1,19 @@
+Verified 0/5 symbols. Waiting for Foo to verify.
+Verified part 1/3 of Foo, located at line 4, using 8,7E+002 resources
```

Comment:
This seems like it may be locale-specific output, using commas in some locales and decimal points in others, so I imagine this test may fail in CI. Or does the `E1` style specify commas?

Reply:
> I imagine this test may fail in CI

Is that based on previous experience? Is there a locale difference between the platforms?

Reply:
Oh, yeah, that's all I was suggesting: ignoring the locale. :)